src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -63,6 +70,27 @@ export const buildTransactionFrom = async (
   const isSendTokenTx = await isTokenTransfer(tx.data, tx.value)
   const customTx = tx.to !== safeAddress && !!tx.data && !isSendTokenTx
 
+  let refundParams = null
+  if (tx.gasPrice > 0) {
+    let refundSymbol = 'ETH'
+    let decimals = 18
+    if (tx.gasToken !== ZERO_ADDRESS) {
+      const gasToken = await (await getHumanFriendlyToken()).at(tx.gasToken)
```

Comment:
@germartinez @mikheevm Is there some utils to get symbol and decimals for a token that check the information in local storage before doing a network request?

Reply:
@rmeissner we currently don't store them in the local storage, we were planning to implement it later